library/platform_util.c

```diff
@@ -133,3 +134,12 @@ struct tm *mbedtls_platform_gmtime_r( const mbedtls_time_t *tt,
 #endif /* _WIN32 && !EFIX64 && !EFI32 */
 }
 #endif /* MBEDTLS_HAVE_TIME_DATE && MBEDTLS_PLATFORM_GMTIME_R_ALT */
+
+#if defined( MBEDTLS_CHECK_PARAMS ) && defined(MBEDTLS_PLATFORM_C) && \
+    defined(MBEDTLS_DEBUG_INVALID_PARAMS)
```

Comment:
`MBEDTLS_DEBUG_INVALID_PARAMS` isn't documented. It seems to me that this definition of `mbedtls_param_failed()` should be removed entirely.

Reply:
I think it's better to remove `MBEDTLS_DEBUG_INVALID_PARAMS` for now.

Reply:
No action needed: as noted above, it *is* documented. Let's leave it here then (at least for now).

Reply:
I might be blind, but it looks like it has been removed in the latest commit.